docs/2024/ci-scanner/index.md

```diff
@@ -15,17 +15,33 @@ SPDX-FileCopyrightText: 2024 Rajul Jha <email.here>
 
 ## Contact info
 
-- [Email](mailto:email.here)
-- [LinkedIn](https://linkedin.com/in/my-user)
+- [Email](mailto:rajuljha49@gmail.com)
+- [LinkedIn](https://linkedin.com/in/rajuljha)
 
 ## Project title
 
 CI Scanner Improvements
 
 ## What's the project about?
 
-Insert Text Here
+To be able to easily and continuously scan packages with
+fossology checks in CI pipelines, a docker image
+(fossology/fossology:scanner) capable of running license checks (using
+nomos or ojo) and keyword and copyright scans is available.
+This project aims to improve the CI Scanner Image in
+various aspects and numerous quality of life improvements, like
+highlighting the exact location of violation, ability to customize the
+keywords used by the scanner, and improving user experience – allow
+whitelisting from a custom location and ability to download and scan
+dependencies
```

Comment:
Please simplify this. Either write it in bullet points or separate the paragraphs. Try active voice.